apps/graphql/src/dataloaders/Itinerariesloader.js

```diff
@@ -34,11 +35,74 @@ export type RouteItem = {|
   +id: string,
 |};
 
+// start - new structure
+
+export type Country = {|
+  +id: string,
+  +name: string,
+  +code: string,
+  +slug: string,
+  +flagURL: string,
+|};
+
+// @TODO rename PriceType
```

Comment:
Base on what you decided when using to `Type` prefix and when not? 
For example, `Country ` is without prefix and `LocationType` is with.
Is there some logic behind this?
Thanks

Reply:
What about creating a new file for types (ItinerariesloaderTypes.js)?
Because all these types are exported which means that they are reuse or will be reused on other places. 
Benefits will be:
1. smaller Itinerariesloader.js file - easiest to read
2. Clearer will be also when I will need to find some type. That I can look to files with `Types` suffix.

What do you think?

Reply:
I agree. 

Btw I used `Type` suffix because I have to distinguish normal `Date` from `Date` defined by me. I think we should add `Type` suffix everywhere for consistency. What do you think?

Reply:
Sure, if it will help then let's do it.
What do you think guys? @JoseDuSV @tbergq @RobinCsl

Reply:
I separated Types to different file and added suffixes to places where they missed